src/module-vsbridge-indexer-core/Console/Command/ResetEsIndexCommand.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * @package  Divante\VsbridgeIndexerCore
+ * @author Nagaraja Kharvi <nagrgk@gmail.com>
+ * @copyright 2019 Divante Sp. z o.o.
+ * @license See LICENSE_DIVANTE.txt for license details.
+ */
+
+namespace Divante\VsbridgeIndexerCore\Console\Command;
+
+use Magento\Framework\App\ObjectManagerFactory;
+use Magento\Framework\Event\ManagerInterface;
+use Magento\Framework\Indexer\IndexerInterface;
+use Magento\Indexer\Console\Command\AbstractIndexerCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\Framework\Exception\LocalizedException;
+
+/**
+ * Class IndexerReindexCommand
+ */
+class ResetEsIndexCommand extends AbstractIndexerCommand
+{
+    /**
+     * RebuildEsIndexCommand constructor.
```

Comment:
change it to `ResetEsIndexCommand constructor`

Reply:
done